GLAA.Web/Views/Shared/DisplayTemplates/SignUpViewModel.cshtml

```diff
@@ -11,35 +11,35 @@
         <td>Principal Authority's email address</td>
         <td class="@Model.EmailAddress?.EmailAddress">@Html.DisplayFor(m => m.EmailAddress.EmailAddress)</td>
         <td class="change-answer">
-            <a asp-controller="SignUp" asp-action="SignUp" asp-route-id="1">Change<span class="visually-hidden"> Principal Authority's email address</span></a>
```

Comment:
These spaces are necessary otherwise a screenreader could interpret this as "ChangePrincipal Authority"

Reply:
Ah, I only trimmed those out to poke drone! will pop them back in!